dist/mjs/interface/interface.d.ts

```diff
@@ -61,6 +67,7 @@ export interface BaseAddImageParams {
     description_url: string;
     avator_url: string;
     circuit_size: number;
+    creator_paid_proof: boolean;
```

Comment:
This one related to B/E changing.

For scalability, we prefer to not use a new boolean param, better it can be a string ImageType type (like the above TaskStatus) and the interface accept the new defined imageType